more-html/hunter-killer.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Hunter Killer</title>
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="../../build/style33.css?b60dae1425af78c9dfb3" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+        <section class="header">
+            <div class="header-name">
+                <h1>Sliven Cinema</h1>
+            </div>
+        </section>
+        <section class="main-body">
+            <div class="nav">
+                <ul>
+                    <li>
+                        <a href="../index.html">This week</a></li>
+                    <li>
+                        <a href="../coming-soon.html">Coming Soon</a></li>
+                    <li>
+                        <a href="../book-ticket.html">Book a ticket</a></li>
+                    <li>
+                        <a href="../about.html">About the cinema</a></li>
+                </ul>
+            </div>
+            <div class="information-block">
+                <img src="../images/hunter-killer.jpg" class="movie-cover" alt="Hunter Killer movie poster">
+                <h2 class="movie-name">
+                    Hunter Killer
+                </h2>
+                <time class="duration">2h 01min</time>
+                <span class="ghost">|</span>
```

Comment:
do we need separate html element for this decorative  element `|`

Reply:
I put a class in case i need to style it.